src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -0,0 +1,265 @@
+  --pf-c-notification-drawer__list-item--m-read--before--Top: calc(var(--pf-c-notification-drawer__list-item--BorderBottomWidth) * -1);
+  --pf-c-notification-drawer__list-item--m-read--before--Bottom: 0;
+  --pf-c-notification-drawer__list-item--m-read--before--BackgroundColor: transparent;
+  --pf-c-notification-drawer__list-item--list-item--m-read--before--Top: 0;
+  --pf-c-notification-drawer__list-item--list-item--m-read--BoxShadow: inset var(--pf-global--BoxShadow--sm-bottom);
+
+  // List item header
+  --pf-c-notification-drawer__list-item-header--MarginBottom: var(--pf-global--spacer--xs);
+
+  // List item header icon
+  --pf-c-notification-drawer__list-item-header-icon--Color: inherit;
+  --pf-c-notification-drawer__list-item-header-icon--MarginRight: var(--pf-global--spacer--sm);
+
+  // List item header title
+  --pf-c-notification-drawer__list-item-header-title--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-notification-drawer__list-item--m-read__list-item-header-title--FontWeight: var(--pf-global--FontWeight--normal);
+
+  // List item description
+  --pf-c-notification-drawer__list-item-description--MarginBottom: var(--pf-global--spacer--sm);
+
+  // List item timestamp
+  --pf-c-notification-drawer__list-item-timestamp--Color: var(--pf-global--Color--200);
+  --pf-c-notification-drawer__list-item-timestamp--FontSize: var(--pf-global--FontSize--sm);
+
+  // Group
+  --pf-c-notification-drawer__group--m-expanded--group--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-notification-drawer__group--m-expanded--group--BorderTopColor: var(--pf-global--BorderColor--100);
+  --pf-c-notification-drawer__group--m-expanded--MinHeight: #{pf-size-prem(300px)};
+
+  // Group toggle
+  --pf-c-notification-drawer__group-toggle--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__group-toggle--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__group-toggle--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__group-toggle--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__group-toggle--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-notification-drawer__group-toggle--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-notification-drawer__group-toggle--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-notification-drawer__group-toggle--OutlineOffset: #{pf-size-prem(-4px)};
+
+  // Group toggle count
+  --pf-c-notification-drawer__group-toggle-count--MarginRight: var(--pf-global--spacer--md);
+
+  // Group toggle icon
+  --pf-c-notification-drawer__group-toggle-icon--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__group-toggle-icon--Color: var(--pf-global--Color--200);
+  --pf-c-notification-drawer__group-toggle-icon--Transition: .2s ease-in 0s;
+  --pf-c-notification-drawer__group--m-expanded__group-toggle-icon--Transform: rotate(90deg);
+
+  display: flex;
```

Comment:
Is this necessary?

Reply:
Yeah, so that the `__header` will stay fixed and `__body` will be the scrollable item if the viewport height is shorter than the overall list of things in the notification drawer. Or do you think we can achieve that without it?